include/jemalloc/internal/emitter.h

```diff
@@ -75,6 +76,12 @@ struct emitter_s {
 	bool emitted_key;
 };
 
+static inline bool
+is_output_json(emitter_t *emitter) {
```

Comment:
Slight preference for `emitter_is_json` or `emitter_outputs_json` or something, since this is nominally public. But not a hill I want to die on if you disagree.

Reply:
Yes, it makes sense to get the "namespace" right. Slightly prefer `emitter_outputs_json`.